cmake/modules/FindEGL.cmake

```diff
@@ -14,14 +14,18 @@
 #
 #   EGL::EGL   - The EGL library
 
+if(CORE_PLATFORM_NAME_LC STREQUAL rbpi)
+    set(_brcmprefix brcm)
+endif()
+
 if(PKG_CONFIG_FOUND)
-  pkg_check_modules(PC_EGL egl QUIET)
+  pkg_check_modules(PC_EGL ${_brcmprefix}egl QUIET)
```

Comment:
Missing space after `${_brcmprefix}` here and on all other changes below.

Reply:
@hudokkow that's correct, if rbpi search for `brcmegl` otherwise for `egl `

Reply:
Doh, my bad! We're looking for `brcmegl`
lgtm.

Sorry for the noise.